sdk/textanalytics/tests.yml

```diff
@@ -4,13 +4,8 @@ stages:
   - template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
     parameters:
       ServiceDirectory: textanalytics
+      TestMode: RECORD # to make sure recordedData is not null but still testing LIVE mode
       Artifacts:
         - name: azure-ai-textanalytics
           groupId: com.azure
           safeName: azureaitextanalytics
-      # resource creation is disabled because we are trying to test the stable endpoint.
-      # Will enable it when service rolls out AAD to all region.
-      DisableAzureResourceCreation: true
-      EnvVars:
-        AZURE_TEXT_ANALYTICS_API_KEY: $(java-textanalytics-test-api-key)
```

Comment:
why deleting the env vars?

Reply:
it for stable endpoint, which can't not delete and then created.